contao-module/endpoints/counts.php

```diff
@@ -3,11 +3,16 @@
 define('TL_SCRIPT', 'counts.php');
 define('TL_MODE', 'FE');
 
-$init = __DIR__ . '/../../../../../../system/initialize.php';
-if(is_file($init)) { // module installed via symlinks from composer vendors
-	require $init;
-} else { // module copied into modules dir
-	require __DIR__ . '/../../../initialize.php';
+foreach ([
+    __DIR__ . '/../../../initialize.php', // system/modules (Contao 3.5 and Contao 4 if installed manually)
+    __DIR__ . '/../../../../../../../system/initialize.php', // Contao 3.5 with Composer
+    __DIR__ . '/../../../../../../system/initialize.php', // Contao 4 with Composer

```

Comment:
That is not working for me, I think it has to be one folder less. Could you verify this?
`__DIR__ . '/../../../../../system/initialize.php', // Contao 4 with Composer`